spec/unit/cds_synchronizer_spec.rb

```diff
@@ -76,22 +76,23 @@
   end
 
   describe '.apply' do
-    let(:applied_update) { create(:taric_update, :applied, example_date: Time.zone.yesterday) }
-    let(:pending_update) { create(:taric_update, :pending, example_date: Time.zone.today) }
+    let(:applied_update) { create(:cds_update, :applied, example_date: Time.zone.yesterday) }
+    let(:pending_update) { create(:cds_update, :pending, example_date: Time.zone.today) }
 
     context 'with failed updates present' do
-      let(:failed_update) { create(:taric_update, :failed, example_date: Time.zone.yesterday) }
+      let(:failed_update) { create(:cds_update, :failed, example_date: Time.zone.yesterday) }
 
       before do
         failed_update
+        allow(TradeTariffBackend).to receive(:service).and_return('uk')
```

Comment:
needs mock `use_cds?` as well presumably?

Reply:
@jebw I'm not sure I agree as lines 79 & 80 are creating a CDSUpdate class so use_cds? would be redundant here.

app/lib/tariff_synchronizer/cds_update.rb 

![image](https://github.com/trade-tariff/trade-tariff-backend/assets/127106895/4e3dda7e-9b0f-4450-8d92-d801a3e7882e)

Reply:
Good point, so just ignore this :+1: